src/component/organism/list/SelectStat.js

```diff
@@ -3,8 +3,7 @@ import PropTypes from 'prop-types';
 import {Text, View, TouchableOpacity} from 'react-native';
 import {txt} from 'Root/config/textstyle';
 import {temp_style} from 'Templete/style_templete';
-import {selectStat} from 'Organism/style_organism';
-
+import {selectStat} from '../style_organism copy';
```

Comment:
import 시에 상대참조보다 절대참조로 가도록 변경 'Organism/style_organism copy'로 변경해야함
```suggestion
import {selectStat} from 'Organism/style_organism copy';
```